.github/issue_template.md

```diff
@@ -0,0 +1,9 @@
+Please be sure to include:
+
+* what operating system you are developing on (linux or macOS)
```

Comment:
"your operating system (Linux or macOS, including the version)"

Reply:
done.